src/connections.rs

```diff
@@ -41,9 +52,11 @@ impl Connection {
     /// Send message to peer using a uni-directional stream.
     pub async fn send_uni(&self, msg: Bytes) -> Result<()> {
         let mut send_stream = self.handle_error(self.quic_conn.open_uni().await).await?;
-        self.handle_error(send_msg(&mut send_stream, msg).await)
+        self.handle_error(send_msg(&mut send_stream, msg.clone()).await)
             .await?;
 
+        trace!("Message sent");
```

Comment:
this log will not be that useful for production code. 
maybe remove it or make it log some more useful info?

Reply:
removed